### ISSUE TITLE
Is it possible to toggle cursor on all visual block selection?

### ISSUE THREAD

**Issue by uwla** [brenton-leighton/multiple-cursors.nvim#74]
First, thanks for the plugin, finally an awesome multi-cursor that actually works and does not freeze like other vim plugins do.

So, is it possible to add cursor to each line in a visual-block selection?

Expected behavior: start `visual-block`, select some lines (i.e, `5j`), and then add invoke this plugin to add a cursor to each line, exiting the visual-block mode and entering the multi-cursor mode.

**Comment by brenton-leighton** [comment]
Yeah I can add a function for that, when I have time. You can also use a count in normal mode when adding a cursor up/down.

**Comment by brenton-leighton** [comment]
@uwla this has been added in #83. Let me know if it's what you wanted.
